source/issues-and-help.txt

```diff
@@ -4,4 +4,51 @@ Issues & Help
 
 .. default-domain:: mongodb
 
-Lorem ipsum
+We are lucky to have a vibrant MongoDB Java community with lots of varying
+experience of using the Java driver. We often find the quickest way to get
```

Comment:
Considering that we actively avoid using "we" in most of our documentation, this seems slightly out of place. That being said I don't actually mind it myself, and I do think it mostly works here (especially if you haven't noticed its absence everywhere else in our docs)

Reply:
Actively avoiding "we" is news to me. Is there a specific reason why?

Reply:
I think it's more that we avoid first-person subjective. Probably worth bringing up in the team chat if it doesn't line up with your own understanding of our style -- I could be wrong!

Reply:
I don't mind being fluid on writing style, but there has to be a good reason. I know that Nathan and Nick specifically endorsed the usage of "we", especially when providing a reader directions, when I asked about preferred pronouns last year. I'm also wondering if this is a recent decision that I wasn't part of as I'm fairly certain our authoring team for Node Driver Guidance Docs has all used "we" in the documentation.

Reply:
I don't think anything has changed on this recently. Probably just a pre-coffee moment for me.